build.sbt

```diff
@@ -483,9 +490,14 @@ lazy val kernel = crossProject(JSPlatform, JVMPlatform)
   .settings(scoverageSettings)
   .settings(sourceGenerators in Compile += (sourceManaged in Compile).map(KernelBoiler.gen).taskValue)
   .settings(includeGeneratedSrc)
-  .jsSettings(commonJsSettings)
+  .jsSettings(commonJsSettings ++ noDottySettings)
   .jvmSettings(commonJvmSettings ++ mimaSettings("cats-kernel"))
-  .settings(libraryDependencies += "org.scalacheck" %%% "scalacheck" % scalaCheckVersion(scalaVersion.value) % "test")
```

Comment:
If you add `.withDottyCompat(scalaVersion.value)` you should be able to keep this dependency when compiling with Dotty too (https://github.com/lampepfl/dotty-example-project#getting-your-project-to-compile-with-dotty)

Reply:
Oh...I missed that!! Cool